.buildkite/pipeline.yml

```diff
@@ -73,10 +73,6 @@ steps:
     notify:
       - github_commit_status:
           context: "Unit Tests"
-      - slack:
-          channels:
-            - "#mobile-apps-tests-notif"
```

Comment:
Just checking that it was intentional (and not a copy/paste typo) that you changed the channel used in the call to `annotate_test_failures` and now send the notification to `#build-and-ship`… while previously it was sent to this other channel?

Reply:
thanks for checking! yup this is intentional. speaking with @pachlava and @tiagomar, we decided to post this on #build-and-ship since that's the channel we're monitoring now - better to have it in one channel rather than multiple. with this, we would immediately know if a build is failing because of tests or other steps from this notification